<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bootloader</class>
 <widget class="QWidget" name="Bootloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_ping">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Ping</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_verify">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Verify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_erase">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Erase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_jump">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Jump</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_write">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Write</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_read">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_filename">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_choosefile">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Partition</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_partition">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>APP_1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>APP_2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>USER_APP</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_debugconsole">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="autoScrollMargin">
      <number>16</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_detect">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
